docs/templates/process-logging.md

```diff
@@ -0,0 +1,292 @@
+# Workspace Process Logging
+
+The workspace process logging feature allows you to log all system-level
+processes executing in the workspace.
+
+> **Note:** This feature is only available on Linux in Kubernetes. There are
+> additional requirements outlined further in this document.
+>
+> This is an Enterprise feature. To learn more about Coder Enterprise, please
+> contact sales.
+
+Workspace process logging adds a sidecar container to workspace pods that will
+log all processes started in the workspace container (e.g., commands executed in
+the terminal or processes created in the background by other processes).  You
+can view the output from the sidecar or send it to a monitoring stack, such as
+CloudWatch, for further analysis or long-term storage.
+
+Please note that these logs are not recorded or captured by the Coder
+organization in any way, shape, or form.
+
+## How this works
+
+Coder uses [eBPF](https://ebpf.io/) (which we chose for its minimal performance
+impact) to perform in-kernel logging and filtering of all exec system calls
+originating from the workspace container.
+
+The core of this feature is also open source and can be found in the
+[exectrace](https://github.com/coder/exectrace) repo on GitHub repo. The
+enterprise component (in the `enterprise/` directory of the repo) is responsible
+for starting the eBPF program with the correct filtering options for the
+specific workspace.
+
+## Requirements
+
+The host machine must be running a Linux kernel >= 5.8 with the kernel config
+`CONFIG_DEBUG_INFO_BTF=y` enabled.
+
+To check your kernel version, run:
+
+```shell
+$ uname -r
```

Comment:
Our shell commands in the rest of our docs exclude the `$`. Can you remove this for now.

Since the $ is often helpful to understand it is a shell command and not a code block, I created an issue that we can solve in our docs rendering on coder.com: https://github.com/coder/coder.com/issues/301.